docs/en/Control_scripts.md

```diff
@@ -179,6 +179,9 @@ helm delete "$name" --purge
 
 This script checks the status of the deployment.
 
+This script receives the same additional command-line arguments as [`check`](#-check):
+* `--tag` – The _Docker Image tag_ of deployed deployment. (In practice you can use some other string that identifies a version of your system to deploy – you will need to process it accordingly in the script.)
+
```

Comment:
This *is* the `check` command. 

The arguments for it are described below (line 192)(189 originally)

Reply:
Fixed.